linkis-orchestrator/linkis-computation-orchestrator/src/main/scala/org/apache/linkis/orchestrator/ecm/ComputationEngineConnManager.scala

```diff
@@ -105,13 +105,13 @@ class ComputationEngineConnManager extends AbstractEngineConnManager with Loggin
         }
       } catch {
         case t: LinkisRetryException =>
-          val taken = ByteTimeUtils.msDurationToString(System.currentTimeMillis - start)
+          val taken = System.currentTimeMillis() - start
           logger.warn(
             s"${mark.getMarkId()} Failed to askEngineAskRequest time taken ($taken), ${t.getMessage}"
```

Comment:
` s"${mark.getMarkId()} Failed to askEngineAskRequest time taken ($taken), ${t.getMessage}"`
->
 s"${mark.getMarkId()} Failed to askEngineAskRequest time taken: $taken ms, ${t.getMessage}"